Color-Permissions.md

```diff
@@ -9,13 +9,15 @@ This works for the following:
 * Sending mails with `/mail`: `essentials.mail.<suffix>`
 * Setting your own nickname with `/nick`: `essentials.nick.<suffix>`
 * Sending chat messages **(requires EssentialsX Chat)**: `essentials.chat.<suffix>`
+* Editing a sign with `/editsign`: `essentials.editsign.<suffix>`
 * Colors on signs **(requires color signs to be enabled)**: `essentials.signs.<suffix>`
 
 Below is a list of valid permissions suffixes. Note that some codes are only available on EssentialsX 2.16.1.169 and
 above:
 
 | Suffix           | Color codes                                       | EssentialsX version |
 |------------------|---------------------------------------------------|---------------------|
+| `rgb` (group)    | `&#RRGGBB`                                        | 2.18.0+             |
 | `color` (group)  | `&0 &1 &2 &3 &4 &5 &6 &7 &8 &9 &a &b &c &d &e &f` | All versions        |
 | `format` (group) | `&l &m &n &o &r`                                  | All versions        |
 | `magic` (group)  | `&k`                                              | All versions        |
```

Comment:
```suggestion
| `color` (group)  | `&0 &1 &2 &3 &4 &5 &6 &7 &8 &9 &a &b &c &d &e &f` | All versions        |
| `format` (group) | `&l &m &n &o &r`                                  | All versions        |
| `magic` (group)  | `&k`                                              | All versions        |
| `rgb`            | `&#RRGGBB`                                        | 2.18.0+             |
```

RGB is closer to a singular permission than a group - negating the named/legacy colours won't stop you using their RGB hex codes.